vue-project/src/views/AboutView.vue

```diff
@@ -30,8 +31,107 @@
     
 </div>
 <div>
-    <!-- 
+     
         Will add site instructions here most likely
-    -->
+    
+
 </div>
-</template>
\ No newline at end of file
+-->
+<div class="about-page">
+    <h1>Project Goal</h1>
+    <p></p>
+    <br>
+    <h1>About Us</h1>
+    <div v-for="member in teamMembers" :key="member.id" class="team-member">
+      <img :src="member.image" :alt="member.name" class="member-image" />
+      <div class="member-details">
+        <h2>{{ member.name }}</h2>
+        <p>{{ member.role }}</p>
+        <p class="member-description">{{ member.description }}</p>
+      </div>
+    </div>
+  </div>
+  <br>
+    <div>
+<FooterFile></FooterFile>
```

Comment:
The footer component shouldn't be copy pasted between every view. Putting it on App.vue will place it on every page.